staging/src/k8s.io/kubectl/pkg/cmd/edit/edit.go

```diff
@@ -43,6 +43,9 @@ var (
 
 		The default format is YAML. To edit in JSON, specify "-o json".
 
+		Shell defaults to /bin/bash if the SHELL environment variable is not set.
```

Comment:
As a user, it is unclear to my why I care about the shell when I am editing.  I think this belongs in the same paragraph that talks about which editor is executed and how to use the KUBE_EDITOR or EDITOR environment variables

We also need to keep in mind the default shell is different on Windows (cmd).

Maybe split that paragraph, and put the new info right next to where it talks about KUBE_EDITOR, something like this:

```
		The edit command allows you to directly edit any API resource you can retrieve via the
		command-line tools. It will open the editor defined by your KUBE_EDITOR, or EDITOR
		environment variables, or fall back to 'vi' for Linux or 'notepad' for Windows. To open the 
                editor, it will use the shell defined by your SHELL environment variable, or fall back to 
                '/bin/bash' for Linux or 'cmd' for Windows.

		You can edit multiple objects, although changes are applied one at a time. The command
		accepts file names as well as command-line arguments, although the files you point to must
		be previously saved versions of resources.
```

What do you think?

Reply:
I think this makes more sense and looks more concise.

Reply:
I will make the required changes. 👍🏾

Reply:
I have made the required changes.